ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -180,7 +178,7 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 			continue
 		}
 
-		filePath := extraManifestPath + "/" + file.Name()
+		filePath := dir + "/" + localExtraManifestPath + "/" + file.Name()
```

Comment:
Using `localExtraManifestPath` here seems fragile; should `GetResourceFiles()` return that, since it knows what directory it was operating on when it created the `files` list?

Reply:
`GetResourceFiles() ` return the binary execution dir and our extra-manifest dir in relative path to it. Is that what u mean ?

Reply:
I guess I'm confused - If GetResourceFiles returns the directory and the files, why do we need to do anything more than `dir + file.Name()` here?  Why do we need to know that they're in `localExtraManifestPath`?

Reply:
cause our extra-manifest in sub directory not in the same directory.

Reply:
See my [related comment](https://github.com/openshift-kni/cnf-features-deploy/pull/801/files#r752551696)